src/main/java/mandel/dictionary/EnglishDictionary.java

```diff
@@ -0,0 +1,46 @@
+package mandel.dictionary;
+
+import com.opencsv.CSVReader;
+import com.opencsv.exceptions.CsvValidationException;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.*;
+
+/**
+ * Reads the englishDictionary file ONCE.
+ */
+public class EnglishDictionary {
+    List<String[]> wordAndDef = new ArrayList<>();
```

Comment:
Private

Reply:
Should I do package-private? I'm getting an error when I try to make it private. I tried making all the methods private too but it still wanted package-private.

Reply:
makes no sense to me that it would cause an error.